<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolPage</class>
 <widget class="QStackedWidget" name="ToolPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11px &quot;Verdana&quot;;
color: rgb(0, 0, 0);</string>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="basePage">
   <widget class="QLabel" name="baseHeadingLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>710</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18px &quot;Verdana&quot;;
font-weight: bold;
color: rgb(122, 171, 32);</string>
    </property>
    <property name="text">
     <string>Tools</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="baseActionsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>710</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="converterBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Number Converter</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="converterDescLbl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>45</y>
       <width>671</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Allows you to convert between different number formats</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="converterPage">
   <widget class="QLabel" name="converterHeadingLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>710</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18px &quot;Verdana&quot;;
font-weight: bold;
color: rgb(122, 171, 32);</string>
    </property>
    <property name="text">
     <string>Number Converter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="converterBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>711</width>
      <height>125</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="converterModhexTxt">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: normal;</string>
     </property>
     <property name="inputMask">
      <string>nn nn nn nn nn nn nn nn nn nn nn nn nn nn nn nn; </string>
     </property>
     <property name="text">
      <string>               </string>
     </property>
     <property name="maxLength">
      <number>47</number>
     </property>
     <property name="cursorPosition">
      <number>47</number>
     </property>
    </widget>
    <widget class="QLabel" name="converterHexLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:normal;</string>
     </property>
     <property name="text">
      <string>Hexadecimal</string>
     </property>
    </widget>
    <widget class="QLabel" name="converterModhexLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:normal;</string>
     </property>
     <property name="text">
      <string>Modhex</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="converterHexTxt">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: normal;</string>
     </property>
     <property name="inputMask">
      <string>nn nn nn nn nn nn nn nn nn nn nn nn nn nn nn nn; </string>
     </property>
     <property name="text">
      <string>               </string>
     </property>
     <property name="maxLength">
      <number>47</number>
     </property>
     <property name="cursorPosition">
      <number>47</number>
     </property>
    </widget>
    <widget class="QPushButton" name="converterHexCopyBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>571</x>
       <y>28</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="converterModhexCopyBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>571</x>
       <y>58</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="converterDecLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:normal;</string>
     </property>
     <property name="text">
      <string>Decimal</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="converterDecTxt">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: normal;</string>
     </property>
     <property name="inputMask">
      <string>00000000000000000000; </string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="cursorPosition">
      <number>20</number>
     </property>
    </widget>
    <widget class="QPushButton" name="converterDecCopyBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>571</x>
       <y>88</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="converterHexLenLbl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: normal;
color: rgb(118, 116, 108)</string>
     </property>
     <property name="text">
      <string>(0 chars)</string>
     </property>
    </widget>
    <widget class="QLabel" name="converterModhexLenLbl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: normal;
color: rgb(118, 116, 108)</string>
     </property>
     <property name="text">
      <string>(0 chars)</string>
     </property>
    </widget>
    <widget class="QLabel" name="converterDecErrLbl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:normal;
color:rgb(221, 97, 99);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="converterActionsBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>185</y>
      <width>711</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QPushButton" name="converterResetBtn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>25</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="converterConvertBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Convert</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>converterHexTxt</tabstop>
  <tabstop>converterModhexTxt</tabstop>
  <tabstop>converterDecTxt</tabstop>
  <tabstop>converterHexCopyBtn</tabstop>
  <tabstop>converterModhexCopyBtn</tabstop>
  <tabstop>converterDecCopyBtn</tabstop>
  <tabstop>converterConvertBtn</tabstop>
  <tabstop>converterResetBtn</tabstop>
  <tabstop>converterBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
